deploy/registration-service.yaml

```diff
@@ -171,6 +171,16 @@ objects:
                     configMapKeyRef:
                       name: registration-service
                       key: twilio.from_number
+                - name: REGISTRATION_WOOPRA_DOMAIN
+                  valueFrom:
+                    configMapKeyRef:
+                      name: registration-service
+                      key: woopra-domain
+                - name: REGISTRATION_SEGMENT_WRITE_KEY
+                  valueFrom:
+                    configMapKeyRef:
+                      name: registration-service
+                      key: segment-write-key
```

Comment:
could we please keep some consistency in the key strings? Please, use either snake or dot case format - as is used in previous keys

Reply:
updated here: fd6538e88bf12f20ad946a8f8d04acebd5acbb02